tensorflow_addons/metrics/cohens_kappa_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Cohen's Kappa Metric."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_addons.metrics import CohensKappa
+from tensorflow_addons.utils import test_utils
+
+@test_utils.run_all_in_graph_and_eager_modes
+class CohensKappaTest(tf.test.TestCase):
+  def test_config(self):
+    kp_obj = CohensKappa(name='cohens_kappa')
+    self.assertEqual(kp_obj.name, 'cohens_kappa')
```

Comment:
Mind doing `test_config` like this?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/metrics_test.py#L641

Reply:
Done